ansible/roles/search_head/tasks/main.yml

```diff
@@ -29,7 +29,7 @@
     - "{{ splunk_attack_range_dashboard }}"
     - "{{ splunk_cim_app }}"
     - "{{ splunk_escu_app }}"
-    - "{{ splunk_python_app }}"
+    - "{{ splunk_python_app }}"
```

Comment:
I see them the same? what type of space is this?

Reply:
I don't know. Should be a normal one.